src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -173,6 +174,17 @@ public void initialize() {
             row.setDisclosureNode(null);
             row.disclosureNodeProperty().addListener((observable, oldValue, newValue) -> row.setDisclosureNode(null));
 
+            // Opens the group's edit window when the group gets double clicked
+            row.setOnMouseClicked((mouseClickedEvent) -> {
+                if (mouseClickedEvent.getButton().equals(MouseButton.PRIMARY)
+                        && (mouseClickedEvent.getClickCount() == 2)) {
+                    GroupNodeViewModel groupToEdit = EasyBind.monadic(row.itemProperty()).getValue();
```

Comment:
Why is there a call to `Easybind.monadic`? To get the selected group, you only need to call `row.itemProperty().getValue()` and that's it. Please change the line accordingly.

Reply:
Good point, fixed in 3ae9885.